apps/new/widget/page/projects/CatalogImport.jsx

```diff
@@ -0,0 +1,193 @@
+const indexer = "https://nearcatalog.xyz/wp-json/nearcatalog/v1";
+const projects = {};
+const query = "";
+
+const CardSkeletonContainer = styled.div`
```

Comment:
What's difference between CatalogProjectCard vs PotlockProjectCard

Reply:
Potlock Project Card has several other things compared to NEAR Catalog one, it has a profile picture, and funding details too.

Reply:
but we don't show the funding details right?
may be can update the potlockCard components to be reused in both places, by passing in same props, that way we can ensure UI consistency atleast?
is there some extra details that we show in either of them?

Reply:
I agree with this -- I do like the smoothness of the Potlock cards

Reply:
I'll create a reuseable component then!